packages/sanity/src/core/studio/screens/RequestAccessScreen.tsx

```diff
@@ -127,7 +129,7 @@ export function RequestAccessScreen() {
       .request<AccessRequest | null>({
         url: `/access/project/${projectId}/requests`,
         method: 'post',
-        body: {note, requestUrl: window?.location.href},
+        body: {note, requestUrl: window?.location.href, type: 'access'},
```

Comment:
Send type to API - @jwoods02 do you know if this is required or will the API just default to type: 'access'?

Reply:
The API will default to access but it's good to send it and be explicit